modules/ROOT/pages/provision-adapter-configure-service-mesh-CRD.adoc

```diff
@@ -80,6 +80,61 @@ spec:
 . Verify the <<adapter_status, adapter status,>> using the following command:
 `asmctl adapter list`
 
+== Reference Kubernetes Secret
```

Comment:
ditto previous edits to same text; does it need to be repeated in both locations?